### PR TITLE
Buyer list - Draft

### PR DESCRIPTION
**Pull request by Poppe-A**
#6729

### CODE REVIEW

File: apps/festival/festival/src/app/dashboard/organization/member/member.component.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  </article>
+  <article *ngIf="org.email">
+    <h5>Email Address</h5>
+    <a [href]="'mailto:' + org.email">{{ org.email }}</a>
+  </article>
+</section>
+
+<section>
+  <h3>Team</h3>
+  <article fxLayout="row wrap" fxLayoutGap="24px grid">
```
Comment (RemcoSimonides) [inline apps/festival/festival/src/app/dashboard/organization/member/member.component.html:21]: `fxLayoutGap="24px grid"` new magic?

File: libs/ui/src/lib/layout/dashboard/dashboard.component.html [hunk 2]
```diff
@@ -6,7 +6,7 @@
     <ng-content select="aside"></ng-content>
   </mat-sidenav>
 
-  <mat-sidenav-content fxLayout="column" [appContainer]="app" (toggle)="sidenav.toggle()">
+  <mat-sidenav-content fxLayout="column" [appContainer]="app" (toggle)="sidenav.toggle()" layout>
```
Comment (RemcoSimonides) [inline libs/ui/src/lib/layout/dashboard/dashboard.component.html:9]: No, not here, please add it in the list-page component next to grid
Like in https://github.com/blockframes/blockframes/pull/8444/files

File: apps/festival/festival/src/app/dashboard/organization/member/member.component.html (hunk 1 above)
Comment (Poppe-A) [reply apps/festival/festival/src/app/dashboard/organization/member/member.component.html]: I pushed before removing it ... Btw, it's not my creation, it exists in differents organization files (title and memeber components)

File: apps/festival/festival/src/app/dashboard/organization/member/member.component.ts [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+import { OrganizationService } from '@blockframes/organization/+state';
+
+@Component({
+  selector: 'festival-marketplace-organization-member',
+  templateUrl: './member.component.html',
+  styleUrls: ['./member.component.scss'],
+  animations: [fade, fadeList('user-card')],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class MemberComponent implements OnInit {
```
Comment (RemcoSimonides) [inline apps/festival/festival/src/app/dashboard/organization/member/member.component.ts:15]: Move this component (and the equivalent in marketplace) to organization lib

**Review by RemcoSimonides — APPROVED** [review]

**Comment by phpgeekfr** [comment]
@Poppe-A please change target to `develop`

**Pull request merged**